test/ring/middleware/defaults_test.clj

```diff
@@ -99,13 +97,11 @@
       (is (= (set (keys (:headers resp)))
              #{"X-Frame-Options"
                "X-Content-Type-Options"
-               "X-XSS-Protection"
                "Strict-Transport-Security"
                "Content-Type"
                "Set-Cookie"}))
       (is (= (get-in resp [:headers "X-Frame-Options"]) "SAMEORIGIN"))
       (is (= (get-in resp [:headers "X-Content-Type-Options"]) "nosniff"))
-      (is (= (get-in resp [:headers "X-XSS-Protection"]) "1; mode=block"))
```

Comment:
Can you add a small unit test to check that if X-XSS-Protection is on, it actually still works? If we haven't removed the key, we should check to ensure it hasn't been broken.

Reply:
Unit test added.